lua/starfall/libs_sh/game.lua

```diff
@@ -134,6 +146,83 @@ if SERVER then
 		util.BlastDamage(instance.entity, instance.player, vunwrap(damageOrigin), math.Clamp(damageRadius, 0, 1500), damage)
 	end
 
+	--- Fires a bullet. Bullet made with this function will not have any tracer, you will have to make them yourself.
+	-- @server
+	-- @param number? damage The damage dealt by the bullet. (1-100)
+	-- @param number? force The force of the bullets. (0-100)
+	-- @param number? distance Maximum distance the bullet can travel.
+	-- @param number? hullSize The hull size of the bullet. (0-10)
+	-- @param number? num The amount of bullets to fire. (1-5)
```

Comment:
Could maybe make it like that ? `damage The damage dealt by the bullet. Default: (1-100)`
Because else would require the user to do some annoying guess work to know the limit, or else adding functions that return the max value for each of those but that feel like adding many functions for not too much ?

Reply:
yeah I suppose that works